### ISSUE TITLE
tag a new release?

### ISSUE THREAD

**Issue by chenrui333** [udoprog/c10t#314]
it has been many many years since last release in 2011, would it be possible to tag a new release any time soon? Thanks!

**Comment by evildeeds** [comment]
Yes it has been a while; and we should acutally then decide on a clear set of milestones to achive a proper release, this far we looesly followed the version of Minecraft we tested c10t with and as I can't find the time to work on c10t it feels like we have always been lagging behind the currently released version of Minecraft
